runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -132,6 +132,14 @@
 	public static final int DEMONIC_GORILLA_PRAYER_SWITCH = 7228;
 	public static final int DEMONIC_GORILLA_DEFEND = 7224;
 	public static final int IMP_DEATH = 172;
+	public static final int THROW_CHINCHOMPA = 7618;
```

Comment:
Did these animation IDs get in by accident?

Reply:
I pulled them from Woox's PR, but off the top of my head I know the Chinchompa and Burst one are used for helping determine if you are using AOE attack